src/control-flow-basics/loops/for.md

```diff
@@ -0,0 +1,16 @@
+---
+minutes: 5
+---
+
+## `for`
+
+The [`for` loop](https://doc.rust-lang.org/std/keyword.for.html) iterates over
+ranges of values:
+
+```rust,editable
+fn main() {
+    for x in 1..5 {
+        println!("x: {x}");
+    }
+}
+```

```

Comment:
The speaker notes (`<details>`) from the original slide both refer to this bit in particular so please move them to this file.